jenkins/Jenkinsfile

```diff
@@ -160,30 +139,35 @@ EOT
         stage('build_rpm'){
           steps {
             container('rpmbuild'){
-              sh script: """
+              sh script: '''
                 cd ~/rpmbuild
-                git clone https://github.com/cloudify-cosmo/cloudify-agent.git SOURCES && cd SOURCES && git checkout ${env.BRANCH_NAME}
-              """, label: 'Copying repo to separate workspace'
+                git clone https://github.com/cloudify-cosmo/cloudify-agent.git SOURCES && cd SOURCES && git checkout ${BRANCH_NAME}
+              ''', label: 'Copying repo to separate workspace'
               sh script: """
                 curl -o ~/rpmbuild/SOURCES/cloudify-agents.spec https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/master/packaging/cloudify-agents.spec ;
                 mkdir -p ~/rpmbuild/SOURCES/packaging/agents/ ;
                 curl -o ~/rpmbuild/SOURCES/packaging/agents/copy_packages.py https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/master/packaging/agents/copy_packages.py ;
                 """, label: 'Prepare rpm-building environment'
-              sh script: """
+              sh script: '''
```

Comment:
This needs at the very least a "set -e" so that it doesn't continue after something fails. As noted in other comments, it may well want set -eux, and as noted elsewhere it may want to set the pipefail option (so that we have one standard invocation everywhere).